### ISSUE TITLE
Add image processing support

### ISSUE THREAD

**Issue by ia3andy** [quarkiverse/quarkus-web-bundler#42]
automatic change format, resize, crop, ...

**Comment by ia3andy** [comment]
We already added some feature in Qute to be able to support this

**Comment by FroMage** [comment]
Oh really? In Qute?

**Comment by ia3andy** [comment]
@FroMage you remember we need some part of the tree to be able to detect the image tag at buildtime. We discussed it at the f2f. AFAI rememebr @mkouba implemented that part.

**Comment by FroMage** [comment]
What tree? What image tag? I'm not following how this relates to image processing.

**Comment by mkouba** [comment]
> @FroMage you remember we need some part of the tree to be able to detect the image tag at buildtime. We discussed it at the f2f. AFAI rememebr @mkouba implemented that part.

Yes, I did. It's the `Template#findNodes(Predicate<TemplateNode>)` method. And here's an example how to use it: https://github.com/quarkusio/quarkus/blob/main/independent-projects/qute/core/src/test/java/io/quarkus/qute/TemplateNodesTest.java#L32-L59

**Comment by ia3andy** [comment]
https://github.com/quarkiverse/quarkus-qute-web/pull/213/
